scripts/installers/uninstall-mac-pkg

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+"""Script to uninstall AWS CLI V2 Mac PKG"""
+import argparse
+import sys
+import os
+from datetime import datetime
+from subprocess import check_output
+from subprocess import CalledProcessError
+from subprocess import PIPE
+
+SCRIPTS_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(SCRIPTS_DIR)
+
+from utils import run
+from utils import BadRCError
+
+
+_PKG_ID = 'com.amazon.aws.cli2'
+
+
+def uninstall():
+    assert _is_installed() is True, 'Could not find AWS CLI installation.'
+    assert os.geteuid() == 0, 'Script must be run as root (with sudo).'
+
+    root_dir = _get_root_dir()
+    list_files = _get_file_list(root_dir)
+    _erase_files(list_files)
+    list_dirs = _get_dir_list(root_dir)
+    _erase_dirs(list_dirs)
+    _forget()
+    return 0
+
+
+def _get_root_dir():
+    lines = run('pkgutil --pkg-info %s /' % _PKG_ID, echo=False).split('\n')
+    volume_line = lines[2]
+    volume = _trim_before_space(volume_line)
+    location_line = lines[3]
+    location = _trim_before_space(location_line)
+    path = os.path.join(volume, location)
+    return path
+
+
+def _trim_before_space(value):
+    return value[value.find(' ') + 1:]
+
+
+def _get_file_list(root):
+    lines = run(
+        'pkgutil --only-files --files %s /' % _PKG_ID, echo=False).split('\n')
+    pkg_file_list = [os.path.join(root, line) for line in lines]
+    extra_files = _read_install_metadata(root)
+    return pkg_file_list + extra_files
+
+
+def _read_install_metadata(root):
+    metadata_path = os.path.join(root, '.install-metadata')
```

Comment:
We should explain why we need to do some of these steps. It seems like there were a lot of gotchas in figuring this out and it would be good to capture why we need to do it.